.github/workflows/flutter.yml

```diff
@@ -48,6 +48,26 @@ jobs:
     - name: Build
       run: flutter build apk
 
+  test-android:
```

Comment:
Also add 
      - 'resources/audio_samples/**'
      - 'resources/test/**'
to `paths`